programs/libraries/debug.sc

```diff
@@ -0,0 +1,129 @@
+// Adds a debug function which is equivalent to the print function, but prints stuff different colors depending on type. Avoids confusion between null and 'null' and between maps, strings, and nbts
```

Comment:
'debug' is a very generic function. While I understand the overal utility of this method, would suggest naming it more specific with something that is more recognizable: pretty_print

Reply:
Ok, I did this